Packages.props

```diff
@@ -22,8 +22,8 @@
 
 		<PackageReference Update="Microsoft.Data.SqlClient.AlwaysEncrypted.AzureKeyVaultProvider" Version="1.1.1" />
 		<PackageReference Update="Microsoft.SqlServer.Management.SmoMetadataProvider" Version="170.12.0" />
-		<PackageReference Update="Microsoft.SqlServer.DacFx" Version="162.1.59-preview" />
-		<PackageReference Update="Microsoft.SqlServer.DacFx.Projects" Version="162.1.37-alpha" />
+		<PackageReference Update="Microsoft.SqlServer.DacFx" Version="162.1.80-preview" />
+		<PackageReference Update="Microsoft.SqlServer.DacFx.Projects" Version="162.1.80-preview" />
```

Comment:
I didn't push this version of projects to nuget.org. Actually not sure how ST is getting it when it's not pushed to nuget.org. please double check with Benjin

Reply:
oh I see you are updating it in the source code. you might not need to if you only need dacfx changes but again make sure you ask Benjin about it

Reply:
From the original DacFx PR, there was a comment saying SqlPlatforms changes require a vbump of both DacFx and Projects Nuget packages - @Benjin can you confirm?

Reply:
I'm about to upload a version of Projects to nuget.org that references this project, so then you can grab that!  I'll ping you once I've done that.